examples/list_people_test.go

```diff
@@ -8,6 +8,50 @@ import (
 	pb "github.com/google/protobuf/examples/tutorial"
 )
 
+func TestWritePersonWritesPerson(t *testing.T) {
+	buf := new(bytes.Buffer)
+	// [START populate_proto]
+	p := pb.Person{
+		Id:    1234,
+		Name:  "John Doe",
+		Email: "jdoe@example.com",
+		Phones: []*pb.Person_PhoneNumber{
+			{Number: "555-4321", Type: pb.Person_HOME},
+		},
+	}
+	// [END populate_proto]
+	writePerson(buf, &p)
+	want := strings.Split(`Person ID: 1234
+  Name: John Doe
+  E-mail address: jdoe@example.com
+  Home phone #: 555-4321
+`, "\n")
+
+	got := strings.Split(buf.String(), "\n")
+	if len(got) != len(want) {
+		t.Errorf(
+			"writePerson(%s) =>\n\t%q has %d lines, want %d",
```

Comment:
True. I was splitting by lines to make it easier to find if there was a mistake somewhere, but this is a small enough string that it isn't necessary. I've made the test much simpler by just comparing the whole string.